spec/uo_cli_spec.rb

```diff
@@ -627,5 +647,18 @@ def select_measures(test_dir, measure_name_list, workflow = 'base_workflow.osw',
       system("#{call_cli} delete --scenario #{bar_scenario}")
       expect((test_directory / 'run' / 'two_building_create_bar' / '2' / 'data_point_out.json').exist?).to be false
     end
+
+    # it 'runs a GHEDesigner sizing' do
```

Comment:
do we want this test to be commented out?

Reply:
Yes, I added a different test in place of it

Reply:
ok..should remove this one then?

Reply:
Yes, I removed this one